packages-ts/gauntlet-terra-contracts/src/lib/inspection.ts

```diff
@@ -8,3 +10,14 @@ export const wrappedComparableLongNumber = (v: any) => {
   if (!v) return '0'
   return toComparableLongNumber(v)
 }
+
+export const getLatestOCRConfig = async (provider: LCDClient, contract: AccAddress) => {
```

Comment:
I don't know if here is the best place for this, but I don't really know where could it go. Add a TODO to find a better place for now